sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -21,9 +20,17 @@ import java.nio.ByteBuffer
  * @property pixelRatio the scale ratio of the style, default the device pixel ratio
  */
 class Style internal constructor(
-  private val styleManagerRef: WeakReference<StyleManagerInterface>,
+  private val styleManagerRef: StyleManagerInterface,
   override val pixelRatio: Float
 ) : StyleInterface {
+
+  private fun getStyleManager(): StyleManagerInterface {
+    if (styleManagerRef == null) {
```

Comment:
sorry, maybe I missed smth, but why uninitialized field throw `MapboxMapMemoryLeakException`?

Reply:
Good catch. we need to reset this reference in onDestroy.